autoinstrumentation/dotnet/version.txt

```diff
@@ -1 +1 @@
-0.4.0-beta.1
```

Comment:
Can someone confirm?

Reply:
I agree with @Kielek, because that would trigger the build and deploy workflow for .NET image. So that v0.5.0 will be available for consumption otherwise e2e test cases will not be able to pull the image for v0.5.0.

Reply:
https://github.com/open-telemetry/opentelemetry-operator/pull/1276